components/activecampaign/sources/common/base.mjs

```diff
@@ -0,0 +1,74 @@
+import activecampaign from "../../activecampaign.app.mjs";
+import constants from "../../common/constants.mjs";
+
+export default {
+  type: "source",
+  dedupe: "unique",
+  props: {
```

Comment:
I think we should remove `type` and `dedupe` from here and specify them in each individual source file.